package.json

```
@@ -4,45 +4,31 @@
     "packages/*"
   ],
   "options": {
-    "mocha": "--full-trace --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js",
+    "mocha": "--full-trace --require @babel/register --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js",
     "mocha_tdd": "--full-trace --watch --require resources/mochaBootload packages/*/src/**/__tests__/**/*-test.js"
   },
   "scripts": {
-    "test": "npm run lint && npm run pretty-check && npm run testonly && npm run check",
-    "testonly": "lerna run build && babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",
-    "t": "npm run testonly",
-    "lint": "eslint src packages/**/src || (printf '\\033[33mTry: \\033[7m npm run lint -- --fix \\033[0m\\n' && exit 1)",
+    "build": "lerna run build",
+    "test": "yarn run lint && yarn run pretty-check && yarn run testonly && yarn run check",
+"testonly": "lerna run build && babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",    "t": "yarn run testonly",
```

I'm uncomfortable with a command `testonly` also running a build. Also this JSON could do with pretty-printing.

 Could we change this:
```suggestion
    "testwithbuild": "lerna run build && yarn run testonly",
    "testonly": "babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",
    "t": "yarn run testwithbuild",
```

---

this is how it works already:
https://github.com/graphql/graphql-language-service/blob/master/package.json#L12

the reason it might need to do this is to ensure downstream dependencies are built before running tests, or else these dependencies might not resolve. this should be ameliorated in travis CI naturally, however, by ensuring that `yarn test` always executes with `yarn build` first. your suggested change would potentially break CI, unless we change `test` to use `testwithbuild` instead of `testonly`

---

Overall, I'm a fan of keeping these scripts as they are already, and revisiting this in another PR

---

Ok 👍